packages/connect/src/base/stun.ts

```diff
@@ -390,15 +390,15 @@ export function intepreteResults(results: RequestWithResponse[]):
  *
  * @param multiAddrs Multiaddrs to use as STUN servers
  * @param socket Node.JS socket to use for the STUN request
- * @param runningLocally set to true when running a local testnet
+ * @param __preferLocalAddress [testing] assume that all nodes run in a local network
  */
 export async function getExternalIp(
   multiAddrs: Multiaddr[] | undefined,
   socket: Socket,
-  runningLocally = false
+  __preferLocalAddress = false
```

Comment:
Logical error: `runningLocally` means no NAT traversal, just plain old TCP to TCP connections, so wrong option.

Only change behavior if `local` addresses should be considered `public`, e.g. in e2e tests with simulated NAT